### PR TITLE
Launch benchmark cluster directly with Docker compose.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This diff replaces `marathon-dcluster` with a `docker-compose.yml` to simplify the benchmark cluster configuration. The Marathon version extraction is simplified to `./version docker`.

Furthermore we abide to `shellcheck -x ci_run_dcluster.sh`.

JIRA issues: MARATHON-8394

### CODE REVIEW

**Comment by jeschkies** [comment]
Benchmark is running on [marathon-benchmarks-nuc/PR-6484](https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/Marathon-Scale-Tests/job/marathon-benchmarks-nuc/view/change-requests/job/PR-6484/).

File: tests/performance/ci_run.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -x -e
```
Comment (ichernetsky) [inline tests/performance/ci_run.sh]: I would recommend using `-o pipefail` and ideally `-u`, if possible.

Comment (jeschkies) [reply tests/performance/ci_run.sh]: Thanks for the feedback.

File: tests/performance/ci_run.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+# Docker Compose cluster configuration.
+echo "Start cluster."
+MARATHON_VERSION=$("$MARATHON_DIR/version" docker)
+
+# Get full path to docker-compose file
+DOCKER_COMPOSE_FILE="$BASEDIR/config/docker-compose.yml"
+
+# Start docker-compose, detached
+export MESOS_VERSION=1.5.1-rc1
```
[the remark this answers is not in the document]
Comment (jeschkies) [reply tests/performance/ci_run.sh]: I will follow up.

**Pull request merged**